playbooks/roles/backups/tasks/main.yml

```diff
@@ -26,8 +26,11 @@
   when: BACKUPS_PROVIDER == 's3'
   tags: ['backups']
 
-- name: Install Azure Storage package for backups on Azure
-  pip: name="azure-storage" state=present
+- name: Install Python packages required for backups on Azure
+  pip: name={{ item.name }} version={{ item.version }}
+  with_items:
```

Comment:
How are we keeping an eye on staying current with versions? Or in other words, when  a new version rolls out that fixes a critical bug, how do we know? I know this is out of scope of the specific commit, but is a larger engineering question that this magic value raises